KMB/make_KMB_info.py

```diff
@@ -637,24 +637,60 @@ def make_default_fmis_category(self, cache):
 
         :param cache: cache for category existence
         """
-        test_cat = None
-        if self.kommunName:
-            test_cat = 'Archaeological monuments in {} Municipality'.format(
-                self.kommunName)
-            if not self.kmb_info.category_exists(test_cat, cache):
-                test_cat = 'Archaeological monuments in {}'.format(
-                    self.kommunName)
-                if not self.kmb_info.category_exists(test_cat, cache):
-                    test_cat = None
+        muni_cat = self.municipal_subcategory(
+            'Archaeological monuments', cache)
 
-        if test_cat:
+        if muni_cat:
             self.needs_place_cat = False
-            self.content_cats.add(test_cat)
+            self.content_cats.add(muni_cat)
         else:
             self.content_cats.add(
-                'Archaeological monuments in %s' % self.landskap)
+                'Archaeological monuments in {}'.format(self.landskap))
             self.content_cats.add(
-                'Archaeological monuments in %s County' % self.lan)
+                'Archaeological monuments in {} County'.format(self.lan))
+
+    def make_default_bbr_category(self, cache):
+        """
+        Set BBR default categories based on municipality.
+
+        Makes a guess for the category name. If not found it defaults to
+        County (Län) category.
+
+        Populates self.content_cats and modifies self.needs_place_cat.
+
+        :param cache: cache for category existence
+        """
+        muni_cat = self.municipal_subcategory('Listed buildings', cache)
+
+        if muni_cat:
+            self.needs_place_cat = False
+            self.content_cats.add(muni_cat)
+        else:
+            test_cat = 'Listed buildings in {} County'.format(self.lan)
+            if self.kmb_info.category_exists(test_cat, cache):
+                self.content_cats.add(test_cat)
+            else:
```

Comment:
Is this something we're going to see? Looks like the structure on Commons for all the possible values of test_cat is complete so it would require some weird / broken values of self.lan, or Gotland (which is simply _Listed buildings in Gotland_).